gazebo_ros2_control_demos/config/cartpole_controller_velocity.yaml

```diff
@@ -8,6 +8,9 @@ controller_manager:
     joint_state_broadcaster:
```

Comment:
do we need these changes in the position.yaml and position launch file ?

Reply:
They are just an example, we only need them if we want the imu on those as well. But it doesn't make sense because afaik the cartpole moves instantaneously, so the IMU doesn't report anything useful.